packages/Config/RectorConfig.php

```diff
@@ -200,6 +201,11 @@ static function (
 
     public function import(string $filePath): void
     {
+        $paths = [$filePath];
+
+        $filesystemTweaker  = new FilesystemTweaker();
+        [$filePath] = $filesystemTweaker->resolveWithFnmatch($paths);
```

Comment:
wouldn't this line - as is - resolve the `*` but picks only the first paths and leaving all others behind?

Reply:
it currently only get single file, I will update to import multiple files in dir.

Reply:
@staabm I updated to imports multiple configs https://github.com/rectorphp/rector-src/pull/4832/commits/eae8bf6743fa7443e6083570877f74263af29cbf